tests/test_main.py

```diff
@@ -0,0 +1,369 @@
+    drama = Genre.objects.create(name="Drama")
+    western = Genre.objects.create(name="Western")
+
+    reeves = Actor.objects.create(first_name="Keanu", last_name="Reeves")
+    johansson = Actor.objects.create(first_name="Scarlett",
+                                     last_name="Johansson")
+    clooney = Actor.objects.create(first_name="George", last_name="Clooney")
+
+    matrix = Movie.objects.create(title="Matrix", description="Matrix movie")
+    matrix.actors.add(reeves)
+    matrix.actors.add(johansson)
+    matrix.genres.add(action)
+
+    batman = Movie.objects.create(title="Batman", description="Batman movie")
+    batman.genres.add(drama)
+    batman.actors.add(clooney)
+
+    titanic = Movie.objects.create(title="Titanic",
+                                   description="Titanic movie")
+    titanic.genres.add(drama, action)
+
+    good_bad = Movie.objects.create(
+        title="The Good, the Bad and the Ugly",
+        description="The Good, the Bad and the Ugly movie",
+    )
+    good_bad.genres.add(western)
+
+    blue = CinemaHall.objects.create(name="Blue", rows=10, seats_in_row=12)
+    vip = CinemaHall.objects.create(name="VIP", rows=4, seats_in_row=6)
+    cheap = CinemaHall.objects.create(name="Cheap", rows=15, seats_in_row=27)
+
+    MovieSession.objects.create(
+        show_time=datetime.datetime(2019, 8, 19, 20, 30),
+        cinema_hall=blue,
+        movie=matrix
+    )
+    MovieSession.objects.create(
+        show_time=datetime.datetime(2017, 8, 19, 11, 10),
+        cinema_hall=cheap,
+        movie=titanic,
+    )
+    MovieSession.objects.create(
+        show_time=datetime.datetime(2021, 4, 3, 13, 50),
+        cinema_hall=vip,
+        movie=good_bad
+    )
+
+    MovieSession.objects.create(
+        show_time=datetime.datetime(2021, 4, 3, 16, 30),
+        cinema_hall=cheap,
+        movie=matrix
+    )
+
+    get_user_model().objects.create_user(username="user1",
+                                         password="pass1234")
+    get_user_model().objects.create_user(username="user2",
+                                         password="pass1234")
+
+    Order.objects.create(user_id=1)
+    Order.objects.create(user_id=1)
+    Order.objects.create(user_id=2)
+
+    Ticket.objects.create(movie_session_id=1,
+                          order_id=1,
+                          row=7,
+                          seat=10)
+    Ticket.objects.create(movie_session_id=1,
+                          order_id=1,
+                          row=7,
+                          seat=11)
+    Ticket.objects.create(movie_session_id=2,
+                          order_id=2,
+                          row=9,
+                          seat=5)
+    Ticket.objects.create(movie_session_id=2,
+                          order_id=2,
+                          row=9,
+                          seat=6)
+
+
+def test_auth_user_models():
+    assert settings.AUTH_USER_MODEL == "db.User"
+
+
+@pytest.mark.django_db
```

Comment:
Not to copy each time this decorator - you could use smth like this:
<img width="456" alt="image" src="https://user-images.githubusercontent.com/48014331/158791135-51fb7143-4954-4c99-8605-18adc27cbc6b.png">